lib/foreman_maintain/cli/base.rb

```diff
@@ -33,10 +33,12 @@ def print_check_info(check)
         puts "#{desc} #{tags}".strip
       end
 
+      def parse(arguments)
```

Comment:
Just out of curiosity, why do we need this method here?

Reply:
Good catch, this is leftover from some of my testing. I will remove it. I wanted to see when the clamp options are parsed to see if we can use the value of `assumeyes` during feature detection. 

BTW I found out the features are instantiated before Clamp parses the CLI options. I we will need the `assumeyes` during a feature init we will need to add parsing this option for ourselves by regexp or something.

Reply:
Initializing the features after clamp is not possible due to `advanced procedure run` (and perhaps other commands in the future) which creates procedure-commands/options dynamically. At the point when the Command is declared the Features are detected/initialized. Then Clamp builds the options and only then it is able to parse CLI according to that options.

Solutions I see:
 - move the --assumeyes to the ForemanMaintain main command (which would move their possition on CLI: `foreman-maintain --assumeyes advanced procedure run`
 - drop dynamic commands/options
 - parse ARGV ourself when needed 

I prefer the last option as it keeps the dynamic CLI and better UX.

Anyway it does not block us now as we don't use assume yes in a feature init.